docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/30-common-configurations.md

```diff
@@ -0,0 +1,423 @@
+For complete parameter documentation, refer to the
+[chart documentation](https://github.com/akuity/kargo/blob/main/charts/kargo/README.md).
+:::
+
+:::info
+For more information on how to apply these configurations, see the
+advanced installation guides for [Helm](10-advanced-with-helm.md) and
+[Argo CD](20-advanced-with-argocd.md).
+:::
+
+## Standard Kubernetes Configuration
+
+For common Kubernetes configurations, like setting a `nodeSelector`, `labels`,
+`annotations`, `affinity`, `tolerations`, etc. Kargo supports both global and
+resource-specific configurations. For example, to add `labels` to all resources
+created by Kargo, the following configuration can be used:
+
+```yaml
+global:
+  labels:
+    label.example.com/key: value
+```
+
+To add `labels` to a specific component (the `kargo-controller` in this example),
+the following configuration can be used:
+    
+```yaml
+controller:
+  labels:
+    label.example.com/key: value
+```
+
+:::note
+For a full list of supported configurations, refer to the
+[Global Parameters](https://github.com/akuity/kargo/blob/main/charts/kargo/README.md#global-parameters)
+or the component-specific parameters in the chart documentation.
+:::
+
+## API Configuration
+
+Kargo supports a number of API-related configurations that can be set at
+installation time. These configurations are used to control the behavior of
+Kargo's API server and its web-based user interface.
+
+:::info
+The sections below outline common configurations for the API server. For a full
+list of supported configurations, refer to the
+[API Parameters](](https://github.com/akuity/kargo/blob/main/charts/kargo/README.md#api).
+
+### API Host
+
+By default, the API server host (i.e. the domain or IP address that the API is
+accessible at) is set to `localhost`. This host is used for generation of
+Ingress resources, certificates, the OpenID Connect issuer and callback URLs,
+and any URLs that are exposed to users.
+
+To configure the API host, set the following configuration:
+
+```yaml
+api:
+  host: kargo.example.com
+```
+
+:::note
+The host is allowed to include a port number, e.g. `kargo.example.com:8080`,
+but should not include a protocol (e.g. `http://` or `https://`) as this is
+automatically inferred from other configuration options.
+:::
+
+### API Service
+
+By default, Kargo will create a `Service` resource for the API server with the
+type `ClusterIP`, which is only accessible within the cluster or through
+[port forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/).
+
+:::caution
+Changing the API server service type can expose the API server to the internet
+in an insecure manner. Refer to the
+[Secure Configuration](../40-security/10-secure-configuration.md) section for
+more information on securing the API server.
+:::
+
+:::info
+Instead of making use of a `Service` resource, you can also expose the API
+server using an [Ingress resource](#api-ingress).
```

Comment:
I could go either way on monospacing technical terms within links, but I've seen you do this in several places, so I'm following you lead. If you disagree, that's fine too.

```suggestion
server using an [`Ingress` resource](#api-ingress).
```